### PR TITLE
[WIP] Support Python 3.7 in Cloudpickle

### PR DESCRIPTION
**Pull request by HyukjinKwon**
This PR targets to support Python 3.7 in Cloudpickle.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ python:
   - "3.4"
   - "3.5"
   - "3.6"
+  - "3.7"
```
Comment (HyukjinKwon) [inline .travis.yml]: Currently blocked by `https://github.com/travis-ci/travis-ci/issues/9552`.

Comment (HyukjinKwon) [reply .travis.yml]: See also https://github.com/travis-ci/travis-ci/issues/9815

Comment (jakirkham) [reply .travis.yml]: Maybe we can start installing Miniconda and create different versioned Python environments for each matrix element?

Comment (HyukjinKwon) [reply .travis.yml]: Looks there are workarounds for it, for instance, https://github.com/travis-ci/travis-ci/issues/9815#issuecomment-402045581 . I don't think this is super urgent and we could just wait for its support though.

**Comment by tomMoral** [comment]
I am currently working on python3.7 support with #189.
I think you can close this one.

**Comment by HyukjinKwon** [comment]
Yea, thx.

**Pull request closed without merging**